### PR TITLE
Fixes#646:Improved UI for smaller screen sizes.

### PR DESCRIPTION
**Pull request by praveenojha33**
**Changes proposed in this pull request**
-Improved UI for smaller screen sizes.
**Surge Link:**
http://deeply-earthquake.surge.sh/
**Screenshots:** 
fixes

**Fixes**#646

**Screenshots:** 
fixes

**Fixes**#646

### CODE REVIEW

**Comment by praveenojha33** [comment]
@singhpratyush please review

File: src/app/feed/feed-header/feed-header.component.scss [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
+  }
+}
 
 .outer-wrapper{
 	display: flex;
 	flex-direction: column;
 	background: #fafafa;
 	border-bottom: 1px solid #ebebeb;
-
+
```
Comment (ParthS007) [inline src/app/feed/feed-header/feed-header.component.scss]: @praveenojha33 Remove whitespace

**Comment by praveenojha33** [comment]
@ParthS007 I have provided the surge link.Please check.

**Comment by ParthS007** [comment]
image
image

**Comment by praveenojha33** [comment]
@ParthS007 
screenshot_2018-02-06-04-12-23-131_com android chrome
l1
screenshot from 2018-02-06 04-25-22

**Comment by praveenojha33** [comment]
@ParthS007 I have checked for the same screen sizes and results were fine. Please tell me what I have done wrong.

**Comment by ParthS007** [comment]
@praveenojha33  I have shared the screenshots of Firefox and you are testing on Chrome. Please Check on Mozilla with 100% zoom also.

**Comment by praveenojha33** [comment]
Thanks for reviewing @ParthS007 I am closing the PR now, I will fix the issue soon.

**Comment by ParthS007** [comment]
@praveenojha33  no need to close this everything is fine, you have to check responsiveness at only two resolutions.

**Comment by praveenojha33** [comment]
@ParthS007 Please check now I have updated the surge link.

**Comment by praveenojha33** [comment]
@ParthS007 Please check now I have updated the surge link.

**Comment by praveenojha33** [comment]
@mariobehling please review

**Pull request merged**